static/katex.less

```diff
@@ -546,7 +544,18 @@
         .vertical-separator {
             display: inline-block;
             margin: 0 -0.025em;
-            border-right: 0.05em solid black;
+            width: 0.05em;
+
+            // Set a min-width so it will display at least one screen pixel.
+            @media screen {
+                min-width: 1px;
+            }
+
+            // Modify min-width for retina displays. Don't force two screen pixels.
+            @media screen and (-webkit-min-device-pixel-ratio: 2.0),
+                screen and (min-resolution: 192dpi) {
+                min-width: 0.5px;
+            }
```

Comment:
Should `.frac-line` have its `min-height` set in a similar fashion?

Reply:
`frac-line` [already has](https://github.com/Khan/KaTeX/blob/master/static/katex.less#L297) that `min-height` set. That was the purpose of PR #931.

Reply:
Thanks for refreshing my memory.  😅